pettingzoo/tests/performance_benchmark.py

```diff
@@ -24,14 +25,59 @@ def performance_benchmark(env):
             if all(env.dones.values()):
                 _ = env.reset()
 
-        if time.time() - start > 5:
-            end = time.time()
+        if time.process_time() - start > test_sample_time:
+            end = time.process_time()
             break
 
     length = end - start
 
     turns_per_time = turn / length
     cycles_per_time = cycles / length
-    print(str(turns_per_time) + " turns per second")
-    print(str(cycles_per_time) + " cycles per second")
-    print("Finished performance benchmark")
+
+    return cycles_per_time, turns_per_time
+
+
+def check_stability(name, arr):
+    # Stability check based on the pyperf standard at
+    # https://pyperf.readthedocs.io/en/latest/cli.html#pyperf-check
+    mean = arr.mean()
+    std = arr.std()
+
+    if (std > mean / 10):
+        # Warn if the standard deviation is greater than 10% of the mean
+        print(f"[WARNING] for {name} Standard deviation is greater than 10% of mean", file=sys.stderr)
+
+    if (arr.min() < mean / 2):
+        # Warn if the minimum is 50% smaller than the mean
+        print(f"[WARNING] for {name} minimum is 50% smaller than the mean", file=sys.stderr)
+
+    if (arr.max() > 1.5 * mean):
+        # Warn if the maximum is 50% greater than the mean
+        print(f"[WARNING] for {name} maximum is 50% greater than the mean", file=sys.stderr)
+
+
+def performance_benchmark(env):
+    print("Starting performance benchmark")
+
+    test_time = 5
+    test_sample_time = 1
+    check_benchmark_stability = True
```

Comment:
I have left these parameters internally and not added them as arguments list so as to not break the current API. However, if added to the argument list, they still will be backward compatible.

Reply:
It should be fine to add these to the arguments list

Reply:
make sure you update the documentation accordingly